package/gluon-config-mode-contact-info/check_site.lua

```diff
@@ -0,0 +1 @@
+need_boolean('owner.obligatory', false)
```

Comment:
every text file should end with a newline character, maybe you can add one here (github also highlights the missing newline with a red icon)